pkg/ee/default-policy-catalog/policytemplates/require-ro-rootfs.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: kubermatic.k8c.io/v1
+kind: PolicyTemplate
+metadata:
+  name: require-ro-rootfs
+spec:
+  title: Require Read-Only Root Filesystem
+  description: | 
+      A read-only root file system helps to enforce an immutable infrastructure strategy;
+      the container only needs to write on the mounted volume that persists the state.
+      An immutable root filesystem can also prevent malicious binaries from writing to the
+      host system. This policy validates that containers define a securityContext
+      with `readOnlyRootFilesystem: true`.
+  category: Best Practices
+  severity: medium
+  visibility: Global
+  policySpec:
+    validationFailureAction: Audit
```

Comment:
it doesn't require anything, it will just whine somewhere in the obscure log where no one will see it.